.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,63 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'docs/**'
+  pull_request:
```

Comment:
we probably should not run on pull_request,

but we could run on new releases

Reply:
Already changed, but we can't find the docker build failure when pull request